### ISSUE TITLE
Document running SQL Server on Mac and Linux

### ISSUE THREAD

**Issue by IagoSRL** [loconomics/loconomics#851]
### Long Description
With the availability of 'SQL Server 2017 for Linux' and using Docker, is should be possible to run local copies of the dev database on any system not just Windows, rather than rely on the one at 'dev.loconomics.com'.
We need to try it and document the process in our doc files, so we can move to use *dev* site only for deployment of *master* of a *dev* branch and stop doing manual edits.

### Useful links (after a quick search)
- Get from Microsoft at official downloads: https://www.microsoft.com/en-us/sql-server/sql-server-downloads Go down to choose platform:
  - The link offer direct access to install it natively at several Linux distros, but review the Docker option too (may be our main recommendation).
  - Docker option provides option for Windows host or Linux host. I suspect the Linux one is smaller and more recommended for use under Linux and Mac; May the Windows Docker be a recommendation too even for Windows machines?
  - For Mac, choose macOS on the left, Docker on the right. It links to official instructions and download.
- Post ["SQL Server Running on a Mac?!" by David Neal](https://dev.to/reverentgeek/sql-server-running-on-a-mac-29e). Includes some **command line instructions** for **official MS sql-cli tool** available at NPM.
- Post ["VS Code on Mac meets SQL Server on Linux (in Docker)" by Aaron Bertrand](https://blogs.sentryone.com/aaronbertrand/vs-code-mac-sql-linux-docker/). Includes instructions for installing Docker and some workarounds for problems (maybe they apply only to early versions, not sure).
- Additionally to the *sql-cli* command line tool, there is a cross platform visual environment for managing SQL Server from Microsoft, but still in early versions, named [SQL Operations Studio](https://docs.microsoft.com/en-us/sql/sql-operations-studio/download) (it's built on top of VSCode; at some article, an addon for VSCode is recommended too as alternative). Just in case no other tool to manage the database is used (like Razor).

*(I did a first search about SQL on Mac actually, but since is about using it with Docker should be close to using it on Linux, more specific articles should exist by sure)*.

### Tasks
- [ ] Review previous links in detail and give it a try
- [ ] Document at repo docs, with links, know issues and workarounds
- [ ] Update docs about getting a database copy for development
- [ ] **(blocked)** Stop accepting manual changes to *dev* site: every back-end developer should use it's own. *This is expected to be possible when using only the new Nodejs back-end, so is blocked by that*. This task should probably be it's own issue.

**Comment by IagoSRL** [comment]
It's not top priority, but at some point @joshdanielson you should give it a try, so we push forward to be really cross platform. @ndarilek don't recall how you set-up SQL on your machine, natively or through Docker, but your experience will be very helpful here.

**Comment by ndarilek** [comment]
You can see a Docker recipe for running SQL Server in the
docker-compose.yml file of the new backend.

**Comment by IagoSRL** [comment]
Thanks @ndarilek, didn't know that was already there but seems obvious now!
Please @joshdanielson assign yourself this issue to try it on Mac and comment any result or problem, when you have a chance for it. May be good to start getting a copy of the new back-end and try to run it using any set-up doc there, since some things may happen automatically already (like setting up Docker and recreating database) and that's all.

**Comment by joshdanielson** [comment]
@IagoSRL will do. @ndarilek may ask for your help when I get to this (may not be this week).

**Comment by ndarilek** [comment]
Sure. FWIW, you've already done most of it. If all you want to do is get
the database running, checking out the new backend and running:

`docker-compose up`

will launch a database. The rest is just accessing the container and
running the commands, or connecting to the database on localhost with
whatever tools you like. I can certainly help with that when the time comes.
